src/components/sidebar.css

```diff
@@ -0,0 +1,21 @@
+.nav-bar-container {
+  border: 1px red dashed;
+  width: 20%;
+  height: 100vh;
+  padding: 20px;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.nav-links-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 5vh;
+}
+
+.logo-container {
+  margin-bottom: 10vh;
+}
```

Comment:
- [ ] Please add the mobile version of your navbar using a hamburger button icon '=' and  make sure if it is functional. It would be nicer if you find the exact icon in the behance design.